### PR TITLE
add guhd.conf and settings improvement

### PR DESCRIPTION
**Pull request by t-mon**

### CODE REVIEW

**Comment by mzanetti** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/165/
Test PASSed.

**Comment by mzanetti** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/166/
Test PASSed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/178/
Test FAILed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/179/
Test FAILed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/180/
Test FAILed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/181/
Test PASSed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/182/
Test FAILed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/183/
Test FAILed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/184/
Test FAILed.

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/191/
Test FAILed.

File: libguh/guhsettings.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+QVariant GuhSettings::value(const QString &key, const QVariant &defaultValue) const
+{
+    return m_settings->value(key, defaultValue);
+}
+
+bool GuhSettings::isRunningAsDaemon()
+{
+    // FIXME: once this class is in the server, use guhcore::runningMode()
+    QString user = qgetenv("USER");
```
Comment (t-mon) [inline libguh/guhsettings.cpp]: @mzanetti I don't know why, but in jenkins this step always fails!

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/200/
Test FAILed.

**Pull request closed without merging**